install-dependencies.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -e
```

Comment:
I think this will break, since we have calls to tools/java/install-dependencies.sh which don't exist.

Reply:
It does not break.